src/pages/document.ejs

```diff
@@ -9,7 +9,21 @@
   <link rel="icon" href="<%= context.config.publicPath +'favicon.ico'%>" type="image/x-icon">
   
   <!-- runtime endpoints config -->
-  <script src="<%= context.config.publicPath + context.config.define.ENDPOINTS_ENV%>"></script>
+  <script type="text/javascript">
+    fetch( '/api/v1/endpoints' )
+      .then(response => {
+        console.log(`RESPONSE: ${response.status},${response.statusText}`);
```

Comment:
Obviously the `console.log` statements here will be removed; but they've been useful.